src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,41 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class LoginPage extends BasePage {
+
+    public static WebDriver driver = null;
+    public static WebDriverWait wait = null;
+
+    // locators
+    By emailField = By.cssSelector("[type = 'email']");
+    By passwordField = By.cssSelector("[type = 'password']");
+    By submitButton = By.cssSelector("button[type='submit']");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement use
```